gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -399,3 +399,7 @@ ds:
   # After the delay is expired, the gateway continue the shutdown process. Any pending request will have a chance to finish gracefully and the gateway will stop normally unless it takes too much time and a SIGKILL signal is sent to the gateway.
 #  delay: 0
 #  unit: MILLISECONDS
+
+# liquibase updates are disabled by default. The Management API will execute the liquibase scripts
```

Comment:
I think this is not the right way to disable liquibase.
IMO, you have to set `management.jdbc.liquibase` to false.
This parameter is read in `AbstractJdbcRepositoryConfiguration`